rust_wtx_bench/src/main.rs

```diff
@@ -0,0 +1,32 @@
+pub mod grpc_bindings;
+
+use wtx::{
+    data_transformation::dnsn::QuickProtobuf,
+    grpc::{GrpcStatusCode, Server, ServerData},
+    http::{
+        server_framework::{post, Router},
+        ReqResBuffer, Request, Response, StatusCode,
+    },
+};
+
+#[tokio::main]
+async fn main() -> wtx::Result<()> {
+    let router = Router::paths(wtx::paths!((
+        "helloworld.Greeter/SayHello",
+        post(say_hello)
+    ),));
+    Server::new(router)
+        .initial_window_len(u32::MAX)
+        .max_concurrent_streams_num(u32::MAX)
+        .max_frame_len(16384)
+        .max_headers_len(4096)
+        .max_recv_streams_num(u32::MAX)
```

Comment:
Let's avoid fine tuning. Those values might be good for this use case but worse for the others. `wtx` should provide good enough parameters by default - we don't do fine tuning in other implementations.

Reply:
Done